content/en/includes/cdaas/rna-install.md

```diff
@@ -1,3 +1,11 @@
+### Kubernetes permissions for the Remote Network Agent
+
+By default, the RNA is installed with full access to the cluster. At a minimum, the RNA needs permissions to create, edit, and delete all KINDs that you plan to deploy with CD-as-a-Service, in all namespaces to which you plan to deploy. The RNA also requires network access to any monitoring solutions or webhook APIs that you plan to forward through it.
```

Comment:
Capitalized `KIND` really looks like https://kind.sigs.k8s.io/, which often capitalized as KIND

could it be "delete all `Kind`s"?

Reply:
HA! That's what I get for copying Stephen's words verbatim without double-checking.